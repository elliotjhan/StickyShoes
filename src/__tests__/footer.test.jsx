import React from "react";
import Footer from "./../components/footer";
import { render, cleanup } from "@testing-library/react";
import renderer from "react-test-renderer";

test(" on initial render", () => {
  render(<Footer />);
  const tree = renderer.create(<Footer />).toJSON();
  expect(tree).toMatchSnapshot();
  cleanup();
});

