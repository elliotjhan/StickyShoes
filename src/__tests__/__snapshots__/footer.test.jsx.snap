// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[` on initial render 1`] = `
<div
  className="row justify-content-around align-items-center footerContainer"
  data-testid="footer"
>
  <div
    className="col text-center"
  >
    <div>
      Â©
      2022
       | stickyshoes.elliotjhan.com | All Rights Reserved
    </div>
  </div>
</div>
`;
